core/src/main/scala/org/apache/spark/sql/execution/ddl.scala

```diff
@@ -407,14 +407,14 @@ class ShowSnappyTablesCommand(session: SnappySession, schemaOpt: Option[String],
   override val output: Seq[Attribute] = {
     if (hiveCompatible) AttributeReference("name", StringType, nullable = false)() :: Nil
     else {
-      AttributeReference("schemaName", StringType, nullable = false)() ::
+      AttributeReference("database", StringType, nullable = false)() ::
```

Comment:
Could you update the class comment to reflect this change?

Reply:
Has been reverted